src/Tribe/Aggregator/Page.php

```diff
@@ -119,6 +119,7 @@ private function __construct() {
 
 		tribe_asset( $plugin, 'tribe-ea-facebook-login', 'aggregator-facebook-login.js', array( 'jquery', 'underscore', 'tribe-dependency' ), 'admin_enqueue_scripts' );
 
+		tribe_notice( 'tribe-aggregator-legacy-import-plugins-active', array( $this, 'notice_legacy_plugins' ), 'type=warning' );
```

Comment:
You've probably weighed this up already ... but is there a case for making this dismissable (or, is there a case where a user might conceivably wish to keep a legacy plugin active and can't deactivate them right away and would like to be rid of this notice)?